### PR TITLE
Setup local.services.yml

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

**Comment by nlisgo** [comment]
We can improve this if we need to allow entirely different local.services.yml for each instance but for now this should work.

File: salt/pillar/journal-cms.sls [hunk 1]
```diff
@@ -9,6 +9,12 @@ journal_cms:
         user: elife_1_0
         password: legacy_cms
 
+    logs:
+        file_path: private://monolog/
```
Comment (giorgiosironi) [inline salt/pillar/journal-cms.sls]: from a first look provisioning the .yml file seems correct, but I fear what happens with the folders like these one (don't know when/where they are created). We'll test this together using a builder Vagrant VM.

Comment (thewilkybarkid) [inline salt/pillar/journal-cms.sls]: Needs quotes?

**Comment by giorgiosironi** [comment]
At the moment (https://github.com/elifesciences/journal-cms/commit/db54db37836de4a824063c978b4e5fa6696a223f), the `private/` folder gets created like this:
```
[10:55:41][elife@dev--journal-cms:/srv/journal-cms]$ ls -ld private/
drwxrwxr-x 2 elife elife 4096 Apr 25 10:50 private/
```
then:
```
UnexpectedValueException: The stream or file "private://monolog/all.json" could not be opened: failed to open stream: "Drupal\Core\StreamWrapper\PrivateStream::stream_open" call failed
```
I'll add a state to create `private/monolog/`, otherwise the log files cannot be written. However if you plan to write anything else in `private/` this ownership is incorrect, as php-fpm won't be able to write in it.

**Comment by giorgiosironi** [comment]
With the latest push, logs are created with the right permissions and they seem to work well.
Incidentally there is an error being logged:
```
[11:15:27][elife@dev--journal-cms:/srv/journal-cms/web]$ tail ../private/monolog/error.json 
{"message":"Unexpected error during import with operation create for rest.resource.ping_rest_resource: The &quot;ping_rest_resource&quot; plugin does not exist.","context":[],"level":400,"level_name":"ERROR","channel":"config_sync","datetime":{"date":"2017-04-25 11:02:49.493793","timezone_type":3,"timezone":"UTC"},"extra":{"referer":"","ip":"127.0.0.1","request_uri":"http://default/","uid":0,"user":"","process_id":16484}}
{"message":"Unexpected error during import with operation create for rest.resource.ping_rest_resource: The &quot;ping_rest_resource&quot; plugin does not exist.","context":[],"level":400,"level_name":"ERROR","channel":"config_sync","datetime":{"date":"2017-04-25 11:08:49.396798","timezone_type":3,"timezone":"UTC"},"extra":{"referer":"","ip":"127.0.0.1","request_uri":"http://default/","uid":0,"user":"","process_id":18009}}
```

**Comment by nlisgo** [comment]
The `private/monolog/` folder should be being created if it doesn't exists on composer install: https://github.com/elifesciences/journal-cms/blob/develop/scripts/composer/ScriptHandler.php#L52-L58

**Pull request merged**